packages/list/test/mwc-list.test.ts

```diff
@@ -1788,5 +1788,32 @@ suite('mwc-list:', () => {
         }
       });
     });
+
+    suite('performance issue', () => {
+      test(
+          'removing a list should not call layout more than once', async () => {
+            let count = 0;
+            const originalLayout = List.prototype.layout;
+            List.prototype.layout = function(update) {
+              originalLayout.call(this, update);
+              count++;
+            };
+
+            const itemsTemplates = new Array(300).fill(0).map(() => listItem());
```

Comment:
Not sure why our IE tests are not running externally, but can we decrease this to 150? Does that still test correctly? IE is timing out and taking too long (>5s) to process 300 components.

Reply:
I believe the issue is related lit-element. It also has a performance issue when adding a lot of elements to the DOM (even in chrome). 
In the profile I've added to this PR you can see the `_enqueueUpdate` that's taking a lot of time to load the fixture.

Reply:
Digging a bit deeper, it seems like lit-html and lit-element are being called a lot of times by the`list` element.
That's a different issue regarding the addition of elements to the list.